package.json

```diff
@@ -64,18 +66,17 @@
         "dom-align": "1.12.3",
         "lottie-web": "5.8.1",
         "memoize-one": "6.0.0",
+        "react": "17.0.2",
```

Comment:
If React is moved into deps, this will be installed in vscode along with its version of React.

Reply:
I see. How do other libraries handle this?
Ant lists react as a devDep and peerDep (https://github.com/ant-design/ant-design/blob/master/package.json)... maybe thats the way to go. Such an approach means you don't need the something like install-peers, but you don't add the overhead of the an additional react version for implementations.